pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
 
 ```
-CREATE OPTIONS:
-    -s|--sandbox    Build a sandbox rather then a read only compressed image
-    -w|--writable   Build a writable image (warning: deprecated due to sparse
-                    file image corruption issues)
-    -f|-F|--force   Force a rebootstrap of a base OS (note: this does not 
-                    delete what is currently in the image, just causes the core 
-                    to be reinstalled)
-    -T|--notest     Bootstrap without running tests in %test section
-    -s|--section    Only run a given section within the bootstrap (setup,
-                    post, files, environment, test, labels, none)
-
+$ sudo singularity shell --writable lolcow/
 ```
 
-You can think of the different options as follows:
+## Converting containers from one format to another
+If you already have a container saved locally, you can use it as a target to build a new container. This allows you convert containers from one format to another.  For example if you had a squashfs container called `production.simg` and wanted to convert it to a writeable ext3 container called `development.img` you could:
 
-**create writable development folder**
 ```
-sudo singularity build --writable container/ Singularity
+$ singularity build --writable development.img production.simg
 ```
 
-**create writable development ext3 image**
-```
-sudo singularity build --writable container.img Singularity
-```
+Similarly, to convert it to a writable directory (a sandbox):
 
-**create production immutable image**
 ```
-sudo singularity build container.simg Singularity
+$ singularity build --sandbox development/ production.simg
 ```
 
-For details about the Singularity flow and use cases, see our [quickstart](/quickstart).
-
-## Checks
-Next, we will look at checks. Checks haven't been robustly developed (yet!) but offer
-an easy way for an admin to define a security (or any other kind of check) to be run on demand
-for a Singularity image. They are defined (and run) via different tags.
+If you omit any options you can also convert your sandbox back to a read-only compressed squashfs image suitable for use in a production environment:
 
 ```
-CHECKS OPTIONS:
-    -c|--checks    enable checks
-    -t|--tag       specify a check tag (not default)
-    -l|--low       Specify low threshold (all checks, default) 
-    -m|--med       Perform medium and high checks
-    -h|--high      Perform only checks at level high
-
-See singularity check --help for available tags
+$ singularty build production2 development/
 ```
+You can convert the three supported container formats using any combination.
 
+Use care when converting writable ext3 images or sandbox directories to the default squashfs file format.  If changes were made to the writable container before conversion, there is no record of those changes in the Recipe file rendering your container non-reproducible.  It is a best practice to build your immutable production containers directly from a Recipe file instead.
```

Comment:
why is Recipe capitalized?

Reply:
I would say Singularity recipe or build recipe

Reply:
Same reason Dockerfile is capitalized in their docs.  https://docs.docker.com/compose/gettingstarted/ .  Whatever we end up calling this thing (I actually prefer Definition file with the suffix .def) I think it should be a proper noun.

Reply:
@GodloveD it's been called Singularity (akin to Dockerfile) since early on. take a look at how Singularity Hub works - it finds a file called Singularity. if you move around the docs, and in all the code, it's Singularity. So you got the right idea! Just wrong word ::P

Reply:
so here it would be appropriate to change to Singularity recipe or just build recipe.